modules/ROOT/pages/installing-on-VM.adoc

```diff
@@ -0,0 +1,100 @@
+
+=== Writing the FCC File
+Copy the following example into a text editor:
+
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+passwd:
+  users:                  #this section lists all users
+    - name: fcos_user     #the first username goes here
+      ssh_authorized_keys:
+        - ssh-rsa AAAA... #replace the line starting from "ssh" with the contents of the fcos_user's ssh public key file.
+----
+
+Save the file with the name `example.yaml`.
+
+==== FCC Syntax
+YAML files must have consistent indentation. Although `fcct` checks for syntax errors, ensure that the indentation matches the above example. Overall, the FCC files must conform to ``fcct``'s schema. For more information on the schema, take a look at the https://github.com/coreos/fcct/blob/master/docs/configuration-v1_0.md[FCC configuration specification].
+
+=== Downloading and Installing FCCT
+`fcct`, the Fedora CoreOS Config Transpiler, is a tool that produces a JSON Ignition file from the YAML FCC file. Using this configuration file, an FCOS machine can be told to create users, create filesystems, set up the network, install systemd units, and more.
+
+Download the latest version of `fcct` and the detached signature from the https://github.com/coreos/fcct/releases[releases page].
+
+Import the http://coreos.com/security/app-signing-key/[CoreOS Application Signing Key] into your GPG keyring to verify the downloaded application:
+
+`gpg --import <key>`
+
+`gpg --verify <fcct detatched signature file> <fcct binary>`
+
+Once the download is verified, change the permissions to make it executable:
+
+`chmod +x <fcct binary>`
+
+Run fcct and pipe the output to a file:
+
+`fcct --input example.yaml > example.ign`
+
+The `example.ign` file is ready to be used by FCOS to boot into a VM.
+
+=== Downloading and Extracting the FCOS VM Image
+Go to the https://getfedora.org/coreos/download/[download page] to retrieve the latest FCOS qcow file.
+
+Extract the file. Assuming that the downloaded file is called `fedora-coreos-qemu.qcow2.xz`, the command would be the following:
+
+
+`unxz fedora-coreos-qemu.qcow2.xz`
+
+Ensure the correct ownership of the file:
+
+`chown fcos_user:kvm fedora-coreos-qemu.qcow2`
+
+TIP: The above example assumes that you have created the fcos_user on your local machine. If you did not, replace fcos_user with the user that will boot FCOS on the local host.
+
+=== Booting FCOS in a VM with the Ignition File
+To boot FCOS with the Ignition file, you must specify the Ignition file to the hypervisor. In the case of QEMU, do so with the `-fw_cfg` parameter. This sets the `opt/com.coreos/config` key in the QEMU firmware configuration device.
+
+For example:
+
+[source,bash]
+----
+qemu-system-x86_64 -machine accel=kvm -m 2048 -cpu host -nographic \
+	-drive if=virtio,file=path/to/fedora-coreos-qemu.qcow2 \
+	-device virtio-rng-pci \
+	-fw_cfg name=opt/com.coreos/config,file=path/to/ignition-config.ign
```

Comment:
It's important to note that absolute file paths must be used here. QEMU cannot resolve relative file paths for these options.

Reply:
Addressed in today's commit.